arch/inst/Zilsd/ld.yaml

```diff
@@ -0,0 +1,40 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: ld
+long_name: Load doubleword to even/odd register pair
+description: |
+  Loads a 64-bit value into registers rd and rd+1. The effective address is obtained by adding
+  register rs1 to the sign-extended 12-bit offset.
+definedBy:
+  anyOf: [Zilsd]
+assembly: ld rd, offset(rs1)
+encoding:
+  match: -----------------011-----0000011
+  variables:
+    - name: rd
+      location: 11-7
+    - name: rs1
+      location: 19-15
+    - name: imm
+      location: 31-20
+access:
+  s: always
+  u: always
+  vs: always
+  vu: always
+
+operation(): |
+  Bits<XLEN> base = X[rs1];
+  Bits<XLEN> offset = $signed(imm);
+  Bits<XLEN> eff_addr = base + offset;
+
+  Bits<64> data = read_memory<64>(eff_addr, $encoding);
+
+  if (rd % 2 == 0 && rd < 31) {
```

Comment:
"rd" is a 5-bit value, so no need to test for "< 31".

I'm not sure if modulus operator is preferred over a mask test, like:
```
(rd & 1) == 0
```
or a straight bit test:
```
rd[0] == 1'b0
```
(modulus is often realized as an integer divide, but smart compilers can figure out a mod 2 is a bit test, so this probably isn't a big deal.)